pyproject.toml

```diff
@@ -0,0 +1,141 @@
+[build-system]
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "drf-jsonschema-serializer"
+version = "2.0.0"
```

Comment:
since we dropped support for things, we'll need to bump the version to 3.0 in the release PR